lib/routes/bilibili/api-interface.d.ts

```diff
@@ -0,0 +1,384 @@
+    text: string;
+}
+interface Button {
+    jump_style: Jumpstyle;
+    jump_url: string;
+    type: number;
+}
+
+interface Common {
+    button: Button;
+    cover: string;
+    desc1: string;
+    desc2: string;
+    head_text: string;
+    id_str: string;
+    jump_url: string;
+    style: number;
+    sub_type: string;
+    title: string;
+}
+interface Additional {
+    common: Common;
+    type: string;
+}
+interface Richtextnode {
+    orig_text: string;
+    rid?: string;
+    text: string;
+    type: string;
+    jump_url?: string;
+    emoji?: Emoji;
+}
+interface Desc {
+    rich_text_nodes: Richtextnode[];
+    text: string;
+}
+interface Pic {
+    height: number;
+    size: number;
+    url: string;
+    width: number;
+}
+interface Opus {
+    fold_action: string[];
+    jump_url: string;
+    pics: Pic[];
+    summary: Desc;
+    title?: string;
+}
+interface Badge {
+    bg_color: string;
+    color: string;
+    icon_url?: string;
+    text: string;
+}
+interface Stat {
+    danmaku: string;
+    play: string;
+}
+interface Pgc {
+    badge: Badge;
+    cover: string;
+    epid: number;
+    jump_url: string;
+    season_id: number;
+    stat: Stat;
+    stat_hidden: number;
+    sub_type: number;
+    title: string;
+    type: number;
+}
+interface Live {
+    badge: Badge;
+    cover: string;
+    desc_first: string;
+    desc_second: string;
+    id: number;
+    jump_url: string;
+    live_state: number;
+    reserve_type: number;
+    title: string;
+}
+interface Common2 {
+    badge: Badge;
+    biz_type: number;
+    cover: string;
+    desc: string;
+    id: string;
+    jump_url: string;
+    label: string;
+    sketch_id: string;
+    style: number;
+    title: string;
+}
+interface Archive {
+    aid: string;
+    badge: Badge;
+    bvid: string;
+    cover: string;
+    desc: string;
+    disable_preview: number;
+    duration_text: string;
+    jump_url: string;
+    stat: Stat;
+    title: string;
+    type: number;
+}
+interface Major {
+    opus?: Opus;
+    type: MajorType;
+    archive?: Archive;
+    pgc?: Pgc;
+    live?: Live;
+    common?: Common2;
+}
+interface Moduledynamic {
+    additional?: Additional;
+    desc?: Desc;
+    major?: Major;
+    topic?: any;
+}
+interface Emoji {
+    icon_url: string;
+    size: number;
+    text: string;
+    type: number;
+}
+
+interface Item {
+    desc: Desc;
+    type: number;
+}
+interface Moduleinteraction {
+    items: Item[];
+}
+interface Threepointitem {
+    label: string;
+    type: string;
+}
+interface Modulemore {
+    three_point_items: Threepointitem[];
+}
+interface Comment {
+    count: number;
+    forbidden: boolean;
+}
+interface Like {
+    count: number;
+    forbidden: boolean;
+    status: boolean;
+}
+interface Modulestat {
+    comment: Comment;
+    forward: Comment;
+    like: Like;
+}
+interface Moduletag {
+    text: string;
+}
+interface Modules {
+    module_author: Moduleauthor;
+    module_dynamic: Moduledynamic;
```

Comment:
```suggestion
    module_author?: Moduleauthor;
    module_dynamic?: Moduledynamic;
```

`module_author` and `module_dynamic` are also optional fields, so maybe they also should be marked with `?`?